src/hooks/utils.c

```diff
@@ -0,0 +1,29 @@
+#include <postgres.h>
+
+#include <catalog/pg_type_d.h>
+#include <nodes/makefuncs.h>
+#include <nodes/pg_list.h>
+
+List *get_operator_oids()
```

Comment:
Can you rename this or prefix with `ldb_` 
At function callsite, I thought this is a postgres native function, and was confused why we wanted all operator oids.

Reply:
Yes, done.